docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+  express:
+      build: .
+      container_name: express
+      ports:
+        - "3004:3004"
+      depends_on:
+        - postgres
+      links: 
+        - postgres
+      command: ["wait-for-it/wait-for-it.sh", "postgres:5432", "--" ,"node", "server/index.js"]
+
+
+  postgres:
```

Comment:
are you planning on running your Postgres database on the same instance as your app?